maplibre_gl_platform_interface/lib/src/location.dart

```diff
@@ -84,6 +84,21 @@ class LatLngBounds {
     return <dynamic>[southwest.toJson(), northeast.toJson()];
   }
 
+  bool contains({required LatLng point}) {
+    final bool isLatitudeInBounds = point.latitude >= southwest.latitude &&
+        point.latitude <= northeast.latitude;
+
+    final bool isLongitudeInBounds;
+    if (southwest.longitude <= northeast.longitude) {
+      isLongitudeInBounds = point.longitude >= southwest.longitude &&
+          point.longitude <= northeast.longitude;
+    } else {
+      isLongitudeInBounds = point.longitude >= southwest.longitude ||
```

Comment:
Those are similar, why the `if`?

Reply:
I think this condition is necessary (correct me if I'm wrong). The `if` statement handles two different cases based on the relationship between `southwest.longitude` and `northeast.longitude`.